<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="fr" sourcelanguage="en">
<context>
    <name>generals</name>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="44"/>
        <source>Show errors</source>
        <translation type="obsolete">Afficher erreures</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="45"/>
        <source>Show warnings</source>
        <translation type="obsolete">Afficher avertissements</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="38"/>
        <source>Create layer</source>
        <translation>Créer layer</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="41"/>
        <source>Cancel</source>
        <translation>Annuler</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="44"/>
        <source>Show error log</source>
        <translation>Afficher le log des erreurs</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="45"/>
        <source>Upload xtf-log file</source>
        <translation>Choisir chemin ou URL d&apos;un fichier XTF-Error-Log</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="46"/>
        <source>Show log for existing layer</source>
        <translation>Afficher le log des erreurs pour un layer existant</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="47"/>
        <source>Only layers created with this plugin can be selected</source>
        <translation>Seuls les layers créés avec ce plugin peuvent être sélectionnés</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="277"/>
        <source>No valid file</source>
        <translation>Fichier non valide</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="267"/>
        <source>Could not get a valid XTF-Log file from specified Url</source>
        <translation>Impossible d&apos;obtenir un fichier XTF-Log valide à partir de l&apos;URL spécifiée</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="277"/>
        <source>No valid XTF-Log file at specified Path</source>
        <translation>Aucun fichier XTF-Log valide au chemin d&apos;accès spécifié</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="42"/>
        <source>Error log</source>
        <translation type="obsolete">XTFLog-Checker log des erreurs</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="413"/>
        <source>No Errors</source>
        <translation>Aucune erreur</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="210"/>
        <source>The selected XTF file contains no Ilivalidator-Errors, select another file.</source>
        <translation>Le fichier sélectionné ne contient aucune erreur du type IliVErrors, sélectionnez un autre fichier.</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="272"/>
        <source>Large file</source>
        <translation>Fichier de plus de 5 MB</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="272"/>
        <source>Processing of large XTF-Log files might take a while</source>
        <translation>Le traitement de fichiers volumineux peut prendre un certain temps</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="100"/>
        <source>Unsupported file</source>
        <translation type="obsolete">Fichier non pris en charge</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="100"/>
        <source>Model {model_name} not supported.</source>
        <translation type="obsolete">Modèle non pris en charge</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="307"/>
        <source>No valid geometry</source>
        <translation>Aucune géométrie valide</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="307"/>
        <source>No Point, Line or Surface Geometries found.</source>
        <translation>Aucune géométrie de point, ligne ou surface trouvée</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="413"/>
        <source>The selected XTF file contains no igCheck-Errors, select another file.</source>
        <translation>Le fichier XTF sélectionné ne contient aucune erreur igCheck, veuillez en sélectionner un autre</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="35"/>
        <source>Show infos</source>
        <translation type="obsolete">Afficher les infos</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="57"/>
        <source>igCheck - Point Errors</source>
        <translation type="obsolete">igCheck – Erreurs de points</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="59"/>
        <source>igCheck - Line Errors</source>
        <translation type="obsolete">igCheck – Erreurs de lignes</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="61"/>
        <source>igCheck - Surface Errors</source>
        <translation type="obsolete">igCheck – Erreurs de surfaces</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="63"/>
        <source>igCheck Error log</source>
        <translation type="obsolete">Journal des erreurs igCheck</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="104"/>
        <source>Unsupported error file version , Model {model_name} not supported yet - please contact developer if you are interested to add support for this model format!</source>
        <translation> Version de fichier d&apos;erreur non prise en charge interlis2.4, ErrorLog24 iG/Check non pris en charge pour le moment - veuillez contacter le développeur si vous êtes intéressé à ajouter la prise en charge de ce format de modèle !</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="120"/>
        <source>Unsupported error file version interlis2.4, ErrorLog24 iG/Check not supported yet - please contact developer if you are interested to add support for this model format!</source>
        <translation> Version de fichier d&apos;erreur non prise en charge interlis2.4, ErrorLog24 iG/Check non pris en charge pour le moment - veuillez contacter le développeur si vous êtes intéressé à ajouter la prise en charge de ce format de modèle !</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="125"/>
        <source>Unsupported error file version interlis2.4, ErrorLog24 IliVErrors not supported yet - please contact developer if you are interested to add support for this model format!</source>
        <translation> Version de fichier d&apos;erreur non prise en charge interlis2.4, Modèle {model_name} / {model_sender} non pris en charge pour le moment - veuillez contacter le développeur si vous êtes intéressé à ajouter la prise en charge de ce format de modèle !</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="127"/>
        <source>Unsupported error file version interlis2.4, Model {model_name} / {model_sender} not supported yet - please contact developer if you are interested to add support for this model format!</source>
        <translation> Version de fichier d&apos;erreur non prise en charge, Modèle {model_name} non pris en charge pour le moment - veuillez contacter le développeur si vous êtes intéressé à ajouter la prise en charge de ce format de modèle !</translation>
    </message>
</context>
</TS>
